src/lib/log.c

```diff
@@ -48,3 +49,68 @@ void log_put(uint8_t c)
 		++buf_idx;
 	}
 }
+
+
+// Destination for logs.
+static enum {
+	log_dst_log,
+	log_dst_stderr
+}
+log_dst = log_dst_stderr;
```

Comment:
This really confused me because the `log_dst = log_dst_stderr;` was on a new line. Please pull this onto the same line as the `}`. Also, according to `CODINGSTYLE.md` please use all upper case for the values.

Reply:
I agree, but the code was indented by uncrustify.  I don't fight tools.

Reply:
tools are there to aid not to dictate. please move it to the line with the `}`. The current form just looks wrong an confusing and style(9) as examples with `typedef structs` which have the more natural way as well.